install.org

```diff
@@ -4,46 +4,53 @@
-~M-x package-install RET org RET~ (see [[file:elpa.html][Org ELPA]]).
-Alternatively you can download the package as a [[https://orgmode.org/org-9.3.7.tar.gz][tar.gz]] or [[https://orgmode.org/org-9.3.7.zip][zip]] archive.
+You can install the latest version of Org as a package by invoking
+~M-x package-install RET org RET~ once you have configured Emacs
+to include [[file:elpa.html][Org ELPA]] as a package repository.
 
-The bleeding-edge development version can be viewed at [[https://code.orgmode.org/bzg/org-mode][code.orgmode.org]] or by
-cloning the repo:
+Alternatively you can download the package as a [[https://orgmode.org/org-9.4.tar.gz][tar.gz]] or [[https://orgmode.org/org-9.4.zip][zip]] archive.
+
+The bleeding-edge development version or Org is available at [[https://code.orgmode.org/bzg/org-mode][code.orgmode.org]].
+You can clone the repo via:
 #+BEGIN_SRC shell
 ~$ git clone https://code.orgmode.org/bzg/org-mode.git
 #+END_SRC
-Daily snapshots of the development version: [[https://orgmode.org/org-latest.tar.gz][tar.gz]] or [[https://orgmode.org/org-latest.zip][zip]].
 
-Worg will tell you more on the [[https://orgmode.org/worg/dev/org-build-system.html][build options]] and on how to [[https://orgmode.org/worg/org-faq.html#keeping-current-with-Org-mode-development][stay on the bleeding edge]].
+Daily snapshots of the development version are released as [[https://orgmode.org/org-latest.tar.gz][tar.gz]] or [[https://orgmode.org/org-latest.zip][zip]] files.
+
+Worg has documentation about [[https://orgmode.org/worg/dev/org-build-system.html][build options]] and about how to [[https://orgmode.org/worg/org-faq.html#keeping-current-with-Org-mode-development][stay on the bleeding edge]].
 
-One will also find a large number of packages to augment Org on [[https://melpa.org/#/?q=org-mode&sort=downloads&asc=false][MELPA]].
+There are also many packages that augment Org available on [[https://melpa.org/#/?q=org-mode&sort=downloads&asc=false][MELPA]].
 
 ** Get started
 
-When you're ready to start trying out Org, take a look at the [[file:quickstart.org][Quickstart]] page
-for an overview of the syntax.
+Once you're ready to Org, check out the [[file:quickstart.org][Quickstart]]
+page for an overview of the syntax.
 
 * Doom, Spacemacs, and popular configs
-The Emacs kits /Doom/ and /Spacemacs/ both come with default setups for Org --- i.e.
-Org and a selection of packages. Here's what you get with [[https://github.com/hlissner/doom-emacs/blob/develop/modules/lang/org/packages.el][Doom]], and likewise
-[[https://github.com/syl20bnr/spacemacs/blob/master/layers/%2Bemacs/org/packages.el][Spacemacs]]. It's worth noting that /Doom/ installs a version of Org close to the
-development ~HEAD~.
+The Emacs starter kits /Doom/ and /Spacemacs/ come with a curated set of packages and configurations that may be
+more familiar to users coming from other editors (e.g. Vim). They include Org along with integrations between
```

Comment:
Minor niggle (I'm a Doom user :stuck_out_tongue:) I feel that Doom/Spacemacs have utility seperate from 'familiarity layers' with other editors.

Reply:
If this comes through as them being only familiarity layers that is definitely not intended. I think I need to add something to the first sentence that captures the fact that they are definitely more than just layers since the mention of curated packages is not sufficient.

Reply:
The way I read it the start was fine "come with a curated set of packages and configurations" but then I found "that may be
more familiar to users coming from other editors" potentially confusing - as (maybe I'm looking too hard?) I saw an interpretation that in Doom/Spacemacs the whole point of the packages/configurations is to make them more familiar to users from other editors.
Shouldn't take more than a small tweak to iron this out though I don't think.

Reply:
Made a bigger change around this due to including Scimax, but also addressed this issue.